common/messages.py

```diff
@@ -55,11 +55,19 @@
 _MSGID_LAUNCHER2GAME_PINGS = 0x4002
 _MSGID_LAUNCHER2GAME_INIT = 0x4003
 
+_MSGID_CLIENT2LOGIN_CONNECT = 0x5000
+_MSGID_CLIENT2LOGIN_SWITCHMODE = 0x5001
+
+_MSGID_LOGIN2CLIENT_MODEINFO = 0x5002
```

Comment:
Make this 0x6000 so we won't have to renumber if there's another client2login message later.

Reply:
DOne